misc/openapi-ballerina/modules/openapi-to-ballerina-generator/src/main/java/org/ballerinalang/openapi/cmd/OpenApiGenServiceCmd.java

```diff
@@ -182,11 +190,21 @@ public void execute() {
             }
         }
 
+        if (output.isEmpty()) {
+            output = executionPath;
+        }
+
         //Set source package for the generated service
         generator.setSrcPackage(moduleArgs.get(0));
 
+        //Set relative path for contract path which will be printed on the generated service bal file
+        Path absPath = Paths.get(resourcePath.toString());
+        Path basePath = Paths.get(output);
+        Path pathRelative = basePath.relativize(absPath);
+
         try {
-            generator.generateService(executionPath, resourcePath.toString(), moduleArgs.get(1), output);
+            generator.generateService(executionPath, resourcePath.toString() , pathRelative.toString(),
```

Comment:
formatting issue additional space between comma and parameter

Reply:
Fixed with commit c1f65eb